packages/emails/email-workflow.ts

```diff
@@ -0,0 +1,288 @@
+// eslint-disable-next-line no-restricted-imports
+import { proxyActivities } from "@temporalio/workflow";
+import type { TFunction } from "next-i18next";
+
+import type { EventNameObjectType } from "@calcom/core/event";
+import type * as activities from "@calcom/emails/email-manager";
+import type { CalendarEvent, Person } from "@calcom/types/Calendar";
+
+import type { MonthlyDigestEmailData } from "./src/templates/MonthlyDigestEmail";
+import type { EmailVerifyLink } from "./templates/account-verify-email";
+import type { OrganizationNotification } from "./templates/admin-organization-notification";
+import type { EmailVerifyCode } from "./templates/attendee-verify-email";
+import type { Feedback } from "./templates/feedback-email";
+import type { PasswordReset } from "./templates/forgot-password-email";
+import type { OrgAutoInvite } from "./templates/org-auto-join-invite";
+import type { OrganizationEmailVerify } from "./templates/organization-email-verification";
+import type { TeamInvite } from "./templates/team-invite-email";
+
+const emailActivities = proxyActivities<typeof activities>({
+  startToCloseTimeout: "3 minutes",
+  retry: {
+    // nonRetryableErrorTypes: ["@calcom/emails/EmailError"],
+    maximumAttempts: 20, //This is just a placeholder value. We should set this to a reasonable value later.
+  },
+});
+
+/* Consideration: For some email workflows (e.g. sendScheduledEmails), it might make sense to re-write them for a better experience and better retry logic. For now, we are just going to use the existing functions which works well as-is.
+ */
+
+/* TODO: The event/task payload for some email workflow needs modification to work properly. For example, sendEmailVerificationLink() needs a language property that is of type TFunction. A function is not serializable, so we need a way to send enough info and then do the language translation in the worker.
+ */
+
+export const sendScheduledEmails = async ({
+  data,
```

Comment:
It's a Temporal pattern to use an object for the Workflow input. I'm using `data` to collect the necessary data, perhaps new properties can be added in the future.

I'd like to mention again that the translation function being passed in most (if not) all of the input needs to be moved from the client to the server (worker) because functions are not serializable data since we're moving to a message-queue system. You won't get a type warning for that because the type checking is correct, but at runtime you'll encounter this error.